app/app.rb

```diff
@@ -0,0 +1,76 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative './models/peep.rb'
+require_relative './models/user.rb'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
```

Comment:
Could the separate resources be handled by different controllers to improve the SRPness of this code?